client-src/src/pages/counselor/inq-summary/inq-summary.html

```diff
@@ -69,19 +69,20 @@ <h2><b>Reminders: </b></h2>
       </ion-row>
     </ion-item-divider>
     <ion-card-content>
-      <ion-card>
+      <ion-card *ngFor="let followUp of currentInqFollowUps; let i = index;">
```

Comment:
Is this part of a sub-section?

Reply:
I don't get your query. Can you please elaborate.

Reply:
Attach UI screenshot for clarity

Reply:
![inquiry summary page](https://user-images.githubusercontent.com/20260001/37408251-afa9494e-27c1-11e8-8414-cf4947838811.jpg)